lib/rules/web/admin_console/4.3/common_ui_elements.xyaml

```diff
@@ -392,10 +398,13 @@ check_resource_details:
   action:
     if_param: current_configuration
     ref: check_current_configuration
+  action:
+    if_param: created_at
+    ref: check_created_at
 check_resource_details_key_and_value:
   element: &key_value
     selector:
-      xpath: //dt[contains(text(),'<key>')]/following-sibling::dd[1][contains(.,'<value>')] | //button[(text()='<key>')]/parent::dt/following-sibling::dd[1][contains(.,'<value>')]
+      xpath: //dt[contains(text(),'<key>')]/following-sibling::dd[1][contains(.,'<value>')] | //button[(text()='<key>')]/parent::dt/following-sibling::dd[1][contains(.,'<value>')] | //button[(text()='<key>')]
```

Comment:
I would say not update this
```
$x("//dt[contains(text(),'Update Strategy')]/following-sibling::dd[1][contains(.,'Custom')] | //button[(text()='Update Strategy')]/parent::dt/following-sibling::dd[1][contains(.,'Custom')] | //button[(text()='Update Strategy')]")
```
Array [ button.pf-c-button.pf-m-plain.co-m-pane__details-popover-button ]